components/Info/Info.tsx

```diff
@@ -1,18 +1,18 @@
 import styles from "../Info/Info.module.css";
 export type InfoProps = {
   firstWord: string;
-  otherWords: string;
-  information: string;
+  restOfPhrase: string;
+  content: string;
 };
 
-function Info({ firstWord, otherWords, information }: InfoProps) {
+function Info({ firstWord, restOfPhrase, content }: InfoProps) {
```

Comment:
I still find these hard to understand. What is it you are trying to display here?